04-database/exercises/sql_stored_procedures.md

```diff
@@ -5,22 +5,13 @@
 
 - What is a stored procedure?
 - Basic structure of a PL/pgSQL function
-  - Input and output arguments
-  - Variables and constants
 - Functions that doesn't return any row
 - Function that returns a table
 - Functions with Security Definer
 - Trigger functions
 - Control structures
-  - IF statement
-  - CASE statement
-  - LOOP statement
-  - WHILE loops
-  - FOR loops
 - Pl/pgSQL and PostGIS
-- Exercises
-  - Create a function that inserts a point into a table.
-  - Create a trigger function that writes a string in a column each time a new point is inserted
```

Comment:
Not sure why those lines where removed, maybe I have a different MarkdownTOC configuration?